.github/workflows/ci.yml

```diff
@@ -89,6 +89,12 @@ jobs:
       - name: Lint
         run: just lint-rust
 
+      - name: Install Cargo Shear
+        run: cargo binstall --no-confirm cargo-shear
```

Comment:
Do we need an extra step? Could we reuse the upper `bins: just, taplo-cli` field of `moonrepo/setup-rust@v1`?